tests/js/client/shell/shell-med-queue-blocked-cluster.js

```diff
@@ -0,0 +1,139 @@
+/* jshint globalstrict:false, strict:false, maxlen: 200 */
+/* global fail, assertEqual, assertTrue, assertFalse, arango */
+
+// //////////////////////////////////////////////////////////////////////////////
+// / @brief timeouts during query setup
+// /
+// / DISCLAIMER
+// /
+// / Copyright 2023 ArangoDB GmbH, Cologne, Germany
+// /
+// / Licensed under the Apache License, Version 2.0 (the "License")
+// / you may not use this file except in compliance with the License.
+// / You may obtain a copy of the License at
+// /
+// /     http://www.apache.org/licenses/LICENSE-2.0
+// /
+// / Unless required by applicable law or agreed to in writing, software
+// / distributed under the License is distributed on an "AS IS" BASIS,
+// / WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// / See the License for the specific language governing permissions and
+// / limitations under the License.
+// /
+// / @author Max Neunhoeffer
+// //////////////////////////////////////////////////////////////////////////////
+
+let jsunity = require('jsunity');
+let arangodb = require('@arangodb');
+let db = arangodb.db;
+let { getEndpointsByType,
+      debugCanUseFailAt,
+      debugClearFailAt,
+      debugSetFailAt,
+    } = require('@arangodb/test-helper');
+      
+function medQueueBlockedSuite() {
+  'use strict';
+  const cn = 'UnitTestsMedQueueBlocked';
+
+  return {
+
+    setUp: function () {
+      // We are using the direct connection to set the failure point here,
+      // since for this particular failure point (block scheduler medium
+      // queue) a reconnect is not possible. Therefore we cannot use
+      // debugFailAt.
+      arango.DELETE_RAW("/_admin/debug/failat");
+      db._drop(cn);
+    },
+
+    tearDown: function () {
+      // We are using the direct connection to set the failure point here,
+      // since for this particular failure point (block scheduler medium
+      // queue) a reconnect is not possible. Therefore we cannot use
+      // debugFailAt.
+      arango.DELETE_RAW("/_admin/debug/failat");
+      require("internal").wait(3);  // give the system time to finish the 
```

Comment:
Should this `wait` be moved into the corresponding test? Otherwise we need to wait 3s for every individual test.

Reply:
mhm yes by now we can actually do it, only one test uses the failurePoints. 👍

Reply:
The problem is that the wait has to be after the failure point has been cleared. So we would have to clean up with a `try ... finally` I can do this...

Reply:
moved to finally.